src/screens/AnatomyReviewScreen.js

```diff
@@ -0,0 +1,188 @@
+import React, { useEffect, useState } from "react";
+import {
+    View,
+    StyleSheet,
+    Text,
+    Image,
+    ScrollView,
+    TouchableOpacity,
+} from "react-native";
+import { useNavigation, useRoute } from '@react-navigation/native';
+
+import IconAD from "react-native-vector-icons/AntDesign";
+import IconO from "react-native-vector-icons/Octicons";
+import { colors } from "../../assets/colors/colors";
+import { client } from "../API/client";
+
+function AnatomyReviewScreen() {
+    const navigation = useNavigation();
+    const route = useRoute();
+
+    const [symptomas, setSymptomas] = useState([]);
+    const [basics, setBasics] = useState([]);
+
+    useEffect(() => {
+        client.getEntries()
```

Comment:
I see the logic the of client.getEntries(), picking specific properties, then storing it in a local state used a lot. since the client is only used to fetch static data from the CDN, it will be much more efficient to to create a context, retrieve the data once, and store it in an object inside the context, and retrieve the specific properties with useContext in different screens. we don't need to call it in every endpoint